.travis.yml

```diff
@@ -2,7 +2,7 @@ os:
   - linux
 language: go
 go:
-  - 1.6
+  - 1.7
```

Comment:
The latter seems less serious given Go's compatibility guarantees. We could depend on something deprecated (e.g. the old context package) but it'll always work.

So I'll leave it at 1.7 for now.

Is there a way in Travis to test multiple go versions?